src/mds/cephfs_features.h

```diff
@@ -41,7 +41,9 @@ namespace ceph {
 #define CEPHFS_FEATURE_NAUTILUS         12
 #define CEPHFS_FEATURE_DELEG_INO        13
 #define CEPHFS_FEATURE_OCTOPUS          13
+#define CEPHFS_FEATURE_PACIFIC          14
 #define CEPHFS_FEATURE_METRIC_COLLECT   14
+#define CEPHFS_FEATURE_QUINCY           15
```

Comment:
I had thought that we were moving away from adding feature bits for specific releases and were going to just test for specific features. @batrick I think you objected to a similar change several months ago for that reason? Maybe I'm misremembering...

Reply:
Yeah...it's coming back to me now.

The problem here is that "release" feature bits like this are ambiguous. We do occasionally backport features and fixes to earlier versions, so this flag then becomes problematic. If we fix the problem this works around in pacific releases, then do we add CEPHFS_FEATURE_QUINCY to that MDS?

Feature flags like this need to describe the feature that is being advertised and then the code should test for that feature.

Reply:
The crux of the issue is that releases should not be feature bits... because they are not features. It's lazy coding (we realize now in hindsight). If we are doing our work correctly, then a client (like the kernel) that supports the latest features should have no problem talking to an MDS running version `n-3` or `n` so long as the feature bits are the same.

@idryomov gets credit for alerting us to how problematic this has been for kernel development.

Reply:
> Feature flags like this need to describe the feature that is being advertised and then the code should test for that feature.

Yes!  This is a bit less clear-cut for purely server side features, but for anything client facing feature bits should absolutely not be tied to releases.